go.mod

```diff
@@ -273,4 +274,6 @@ replace github.com/golang/glog => github.com/jdolitsky/glog v0.0.0-2022072917223
 
 // Bring in the latest cosign so we can pass various keys in to verification
 // functions
-replace github.com/sigstore/cosign => github.com/sigstore/cosign v1.13.2-0.20221209171251-6cb723fc299c
+replace github.com/sigstore/cosign/v2 => github.com/vaikas/cosign/v2 v2.0.0-20221215003938-48e6404789cc
+
+// replace github.com/sigstore/cosign => github.com/sigstore/cosign v1.13.2-0.20221209171251-6cb723fc299c
```

Comment:
can we remove this line ?

Reply:
I'll wait until upstream fix goes in, I just wanted to test against my branch to make sure this PR is ready to go as soon as the upstream merges, I'll just rebase to that.